src/snowflake/cli/_plugins/nativeapp/manager.py

```diff
@@ -577,7 +576,10 @@ def create_app_package(self) -> None:
             )
 
     def _expand_script_templates(
-        self, env: jinja2.Environment, jinja_context: dict[str, Any], scripts: List[str]
+        self,
+        render_from_file: Callable[[Path, Dict[str, Any]], str],
```

Comment:
This change made it easier to use `SqlTemplateEnv` (new class joining two jinja envs) rendering and "ordinary" jinja rendering, which was already implemented in `api.rendering.jinja`